### PR TITLE
Add how to access the .profile file

### PR DESCRIPTION
**Pull request by DrBeta**
.profile is technically .bash_profile , so I changed it to that also

### CODE REVIEW

**Comment by DrBeta** [comment]
Can anyone answer?

File: source/using/getting-started.html.md [hunk 1]
```diff
@@ -52,16 +52,15 @@ If you encounter any problems during installation, please visit [this](https://g
 If you do *not* want to grant RubyGems admin privileges for this process, you can
 tell RubyGems to install into your user directory by passing either the
 `--user-install` flag to `gem install` or by configuring the RubyGems environment.
-The latter is in our opinion the best solution. To do this, create or edit the
-`.profile` file in your home directory and add or amend it to include these lines:
+The latter is in our opinion the best solution. To do this open up terminal and type `nano .bash_profile`. Then enter these lines into the file:
```
Comment (dnkoutso) [inline source/using/getting-started.html.md]: there is no need to be specific on `nano` just keep it as `create or edit` your `.bash_profile`

**Comment by DrBeta** [comment]
Can anyone check this?

**Comment by amorde** [comment]
Thanks @DrBeta!

**Comment by DrBeta** [comment]
Thank You

**Pull request merged**